SpatialGDK/Source/SpatialGDKEditor/Private/SpatialGDKEditorSettings.cpp

```diff
@@ -3,31 +3,66 @@
 
 USpatialGDKEditorSettings::USpatialGDKEditorSettings(const FObjectInitializer& ObjectInitializer)
 	: Super(ObjectInitializer)
+	, bDeleteDynamicEntities(true)
+	, bGenerateDefaultLaunchConfig(true)
+	, bStopSpatialOnExit(false)
 {
 	SpatialOSDirectory.Path = GetSpatialOSDirectory();
-	SpatialOSLaunchConfig = GetSpatialOSLaunchConfig();
+	SpatialOSLaunchConfig.FilePath = GetSpatialOSLaunchConfig();
```

Comment:
Curious - did this change how that value was viewed in the editor?

Reply:
Yep, no you get a file picker which makes it easier to select a file